datafusion/execution/src/cache/cache_manager.rs

```diff
@@ -59,6 +75,13 @@ impl CacheManager {
         if let Some(lc) = &config.list_files_cache {
             manager.list_files_cache = Some(Arc::clone(lc))
         }
+        if let Some(mc) = &config.file_metadata_cache {
```

Comment:
Here the `file_metadata_cache` is assigned to `DefaultFilesMetadataCache` if not provided. This makes it easier to enable metadata caching using just `ParquetReadOptions` or `set datafusion.execution.parquet.cache_metadata = true`.